cypress/integration/about.ts

```diff
@@ -0,0 +1,57 @@
+describe('The about page', () => {
+  it('has working link to comparator page', () => {
+    cy.visit('/')
+
+    cy.contains('Comparator').click()
+
+    cy.url().should('include', '/')
+  })
+})
+
+describe('The about page', () => {
+  it('has correct page title', () => {
+    cy.visit('/about')
+
+    cy.title().should(
+      'equal',
+      'About | Octoclairvoyant: Compare GitHub changelogs across multiple releases'
+    )
+  })
+})
+
+describe('The about page', () => {
+  it('contains images which all have a alt attribute', () => {
+    cy.visit('/about')
+    cy.get('img').each(($el) => {
+      cy.wrap($el).should('have.attr', 'alt')
+    })
+  })
+})
+
+describe('The about page', () => {
+  it('contains correct hero text', () => {
+    cy.visit('/about')
+    cy.contains('h1', 'Compare GitHub changelogs across multiple releases')
```

Comment:
Would be better to import the `BRIEF_DESCRIPTION`  from `common.ts` here as well... 😄

Reply:
Aah, that's why you want to import `BRIEF_DESCRIPTION`. I disagree with important that. Better to have hardcoded values within tests 😄 Otherwise we might introduce some error to `BRIEF_DESCRIPTION` and we wouldn't realize because the test would be comparing against the same const, so you can only realize about that hardcoding the expected value in tests.

Reply:
>  Otherwise we might introduce some error to `BRIEF_DESCRIPTION` and we wouldn't realize because the test would be comparing against the same const

Let's keep using the current hard-coded value then.